chaste_codegen/templates/Shared/hpp/includes

```diff
@@ -1,4 +1,6 @@
 #include "ChasteSerialization.hpp"
 #include <boost/serialization/base_object.hpp>
+{% if modifiers|length > 0 %}#include "AbstractCardiacCellWithModifiers.hpp"
+#include "AbstractModifier.hpp"{%- endif %}
 {% if dynamically_loadable %}#include "AbstractDynamicallyLoadableEntity.hpp"{%- endif %}
```

Comment:
I suspect the issue arises when both tests are false. We get a blank line for each false option. Does this approach help?
```
{% if modifiers|length > 0 %}#include "AbstractCardiacCellWithModifiers.hpp"
#include "AbstractModifier.hpp"{%- endif %}
{%- if dynamically_loadable %}#include "AbstractDynamicallyLoadableEntity.hpp"{%- endif %}
```

Otherwise having the opening `if` on the previous line might help? 
```
#include <boost/serialization/base_object.hpp>{% if modifiers|length > 0 %}
#include "AbstractCardiacCellWithModifiers.hpp"
#include "AbstractModifier.hpp"{%- endif %}{% if dynamically_loadable %}
#include "AbstractDynamicallyLoadableEntity.hpp"{%- endif %}
```

Or maybe the following. Would remove an existing blank line I think, but should stop them reappearing?
```suggestion
{%- if modifiers|length > 0 -%}
#include "AbstractCardiacCellWithModifiers.hpp"
#include "AbstractModifier.hpp"
{%- endif -%}
{%- if dynamically_loadable -%}
#include "AbstractDynamicallyLoadableEntity.hpp"
{%- endif -%}
```

Reply:
yes this one is easy to resolve indeed as you suggest